packages/bsky/tests/views/thread.test.ts

```diff
@@ -368,4 +371,86 @@ describe('pds thread views', () => {
       )
     })
   })
+
+  describe('postgates', () => {
```

Comment:
think this might be fit better in `posts.test.ts` instead 🤔 

tho idk I don't hate having the test in the context of `getPostThread` as well, but I am kinda itching for at least one test on the simplest posts route which is `getPosts`